Dynamic Macros on Ergodox infinity not working

When I try to make ergodox infinity firmware with dynamic macros enabled I get the following error:

`./quantum/dynamic_macro.h: In function 'dynamic_macro_led_blink':`

`./quantum/dynamic_macro.h:53:5: error: implicit declaration of function '_delay_ms' [-Werror=implicit-function-declaration]`
`     _delay_ms(100);`
`     ^`
`cc1: all warnings being treated as errors`

I followed the instructions in the [docs](https://docs.qmk.fm/dynamic_macros.html) step by step. Don't think I  missed anything there

--- a/quantum/dynamic_macro.h
+++ b/quantum/dynamic_macro.h
@@ -50,7 +50,7 @@ void dynamic_macro_led_blink(void)
 {
 #ifdef BACKLIGHT_ENABLE
     backlight_toggle();
-    _delay_ms(100);
+    wait_ms(100);
     backlight_toggle();
 #endif
 }

